charts/tekton-pipelines/values.yaml

```diff
@@ -366,6 +438,43 @@ buildpacks:
 kaniko:
   # -- should we enable the kaniko pipeline
   enabled: false
+  postDeployTaskSteps:
+    - name: argo-events
```

Comment:
Doesn't it make sense to describe a task and reuse it instead of copy-paste?

Reply:
done